### PR TITLE
DIG-1409: Add healthcheck.py for docker healthcheck fix

### PR DESCRIPTION
**Pull request by lilyyangyi301**
- Added the required python healthcheck script for the healthcheck fix.

### CODE REVIEW

File: candig_federation/healthcheck.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        f"{keycloak_url}/auth/realms/candig/protocol/openid-connect/token",
+        data=payload,
+    )
+    if response.status_code == 200:
+        return response.json()["access_token"]
+
+def perform_healthcheck():
+    auth_token = get_token(username="user2", password=password, client_id="local_candig", client_secret=client_secret)
+    headers = {"Authorization": f"Bearer {auth_token}"}
+    url = "http://candig.docker.internal:5080/federation/v1/service-info"
```
Comment (OrdiNeu) [inline candig_federation/healthcheck.py]: Need to swap this to use `{FEDERATION_SERVICE_URL}/v1/service-info`, in the same manner that you added KEYCLOAK_URL.

**Review by OrdiNeu — CHANGES_REQUESTED** [review]
One quick change here, otherwise looks good :heavy_check_mark:

File: candig_federation/healthcheck.py (hunk 1 above)
Comment (lilyyangyi301) [reply candig_federation/healthcheck.py]: Fixed.

**Comment by lilyyangyi301** [comment]
Closed because no longer needed.

**Pull request closed without merging**